src/base/bittorrent/torrenthandle.h

```diff
@@ -462,6 +462,8 @@ namespace BitTorrent
 
         bool m_pauseAfterRecheck;
         QHash<QString, TrackerInfo> m_trackerInfos;
+
+        bool m_started = false;
```

Comment:
I cannot easily link started == not paused, or maybe just `m_paused`?

Reply:
"started" doesn't mean "not paused". It means "all initialization has completed and torrent has started regular processing". When torrent added/restored in "paused" state it become "started" immediately after construction (see TorrentHandle constructor). When it is added/restored in "resumed" state, it become "started" after it is really resumed (i.e. after receiving torrent_resumed_alert).

Reply:
I see now, but "started" is a bit vague, other terms like "ready" or "initialized" isn't that suitable here I guess, hope someone can suggest something better.
I would hope for some comments in code at least.

Reply:
>Is my previous comment intended to be the code comment?

Anything is better than nothing.

>And where is the best place for such comment?

Just here?

Reply:
>Just here?

It's just variable declaration here. How can it confuse in this place?.. I will add more descriptive comment on its usage in constructor.